Sparcl nb and readme in tutorials/database/ folder for dr1 branch

stephjuneau opened the pull request:
This PR is for a new notebook for using SPARCL at NERSC to access DESI DR1 spectra and a brief README.md file in that same folder to explain what the notebooks are now there are two different ones.

stephjuneau committed:
New notebook showing How-to-use-SPARCL-at-NERSC

This version is for DESI collaboration members. We will rewrite for the public once DR1 is published.

stephjuneau committed:
Create README.md

Initial README file to get us started. We could improve it when we modify the notebook to their final public version.

weaverba137 commented:
Thank you for adding the `README`. I was thinking that might be helpful. I want to test the notebook again before merging. I probably won't have time for that today.

stephjuneau committed:
Revised nb with updated plots that use mask

Mostly a couple of cosmetic changes and improving the plots by using the mask to omit the masked values.

stephjuneau committed:
Small fixes to nb

Updated date and small things and fresh kernel with clear outputs

stephjuneau commented:
OK, now I'm happy with this version. Cleaned up a bit the plots by using the mask. Ben is busy this morning so I'd like to know if someone else might like to take this new notebook for a spin? @forero or @sbailey or @moustakas ? No worries if y'all are too busy, I'll also post a version in the NERSC filesystem before the ECS tutorial.

weaverba137 reviewed (approved):
I ran the notebook at NERSC. Everything was fine. I have one modification to suggest for the `pip` command, but otherwise this can be merged.

weaverba137 commented on database/How_to_use_SPARCL_at_NERSC.ipynb:
```diff
@@ -0,0 +1,992 @@
+   "outputs": [],
+   "source": [
+    "## Uncomment the following only if SPARCL client is not already installed\n",
+    "#!pip install --upgrade sparclclient"
```
Add `--user` to the `pip` command.

stephjuneau replied:
OK, just added and committed the change

stephjuneau committed:
fixed pip install command

stephjuneau commented:
Hi @weaverba137 -- just made the change to add `--user` to the pip install command.

weaverba137 commented:
Merge when ready.
